### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-[TOC]
-
-# Introduction {#Introduction}
+# Brion
```
Comment (hernando) [inline README.md]: Have you verified how does this look in Doxygen? I think that with your change the project name appears twice.

File: README.md [hunk 2]
```diff
@@ -59,19 +57,19 @@ The higher level library is called Brain and it provides:
...
 
 Building from source is as simple as:
 
     git clone https://github.com/BlueBrain/Brion.git
     mkdir Brion/build
     cd Brion/build
-    cmake ..
-    make
+    cmake -GNinja ..
+    ninja
```
Comment (hernando) [inline README.md]: Ninja is not known by everybody, I'd keep it as it was.

**Comment by eile** [comment]
Both of these changes should have been done and discussed during the last release.

**Comment by hernando** [comment]
If all the rest are the same, then OK

**Pull request merged**